Description: Fix FTBFS with XMMS2 support enabled
 Conky FTBFS when compiled with --enable-xmms2; this patch fixes this issue.
Origin: upstream, http://git.omp.am/?p=conky.git;a=patch;h=8dd9b91bb2751abccc02b53cefd0ab8f0e3b957d
Bug-Debian: http://bugs.debian.org/579893

--- a/src/xmms2.c
+++ b/src/xmms2.c
@@ -223,7 +223,7 @@ int handle_playback_state_change(xmmsv_t *value, void *p)
 				break;
 			case XMMS_PLAYBACK_STATUS_STOP:
 				strncpy(ptr->xmms2.status, "Stopped", text_buffer_size - 1);
-				ptr->xmms2.elapsed = ptr->xmms2.progress = ptr->xmms2.percent = 0;
+ 				ptr->xmms2.elapsed = ptr->xmms2.progress = 0;
 				break;
 			default:
 				strncpy(ptr->xmms2.status, "Unknown", text_buffer_size - 1);
@@ -232,7 +232,7 @@ int handle_playback_state_change(xmmsv_t *value, void *p)
 	return TRUE;
 }
 
-int handle_playlist_loaded(xmmsv_t *value, void *p) 
+int handle_playlist_loaded(xmmsv_t *value, void *p)
 {
 	struct information *ptr = p;
 	const char *c, *errbuf;
